extensions/azurePublish/src/components/azureProvisionDialog.tsx

```diff
@@ -184,17 +184,6 @@ const DialogTitle = {
   },
 };
 
-const getResourceRegion = (item: ResourcesItem): string => {
```

Comment:
moved this to where we map the [selected resources](https://github.com/microsoft/BotFramework-Composer/pull/8562/files#diff-8f3771862f56b8bb7763646529ce7e9cca96d9350d261502f194efcf10e55a56R1143) to consolidate the region logic in one place